.gitignore

```diff
@@ -18,3 +18,12 @@ packages/**/dist
 packages/**/esm
 
 published.txt
+
+# Yarn Files with Zero-Installs
+.yarn/*
+# !.yarn/cache
```

Comment:
Ignores yarn cache folder for now due to files changed becoming clogged with cache files, this will be uncommented in a future PR to make use of zero-installs feature.